python/port/port.h

```diff
@@ -39,6 +41,31 @@ void deinit();
 void registerScriptProvider(ScriptProvider * s);
 void collectRootsAtAddress(char * address, int len);
 
+class ColorParser {
+  private:
+  class NameColorPair {
+  public:
+    constexpr NameColorPair(const char * name, KDColor color) :
+      m_name(name),
+      m_color(color)
+    {}
+    const char * name() const { return m_name; }
+    KDColor color() const { return m_color; }
+  private:
+    const char * m_name;
+    KDColor m_color;
+  };
+
+  public:
```

Comment:
Idea:
```
enum class ColorModes {
   MaxIntensity1 = 1,
   MaxIntensity255 = 255,
   };
```
and replace magic number 255 by `(int)ColorModes::MaxIntensity255` ?

Reply:
Even better : use a static_cast instead of a C-style cast 😄